_posts/2016-xx-xx-release-0.16.md

```diff
@@ -119,6 +119,116 @@ If you are using the low-level qcprot algorithm your self intead of our provided
 wrappers you have to change your code since the API has changed. For more see
 the [CHANGELOG].
 
+## MemoryReader: Reading trajectories from memory
+
+MDAnalysis typically reads trajectories from files on-demand, so that it can efficiently deal with large trajectories - even those that do not fit in memory. However, in some cases, both for convenience and for efficiency, it can be an advantage to work with trajectories directly in memory. In this release, we have introduced a MemoryReader, which makes this possible.
+
+The MemoryReader works with numpy arrays, using the same format as that used by for instance `DCDReader.timeseries()`. You can create a Universe directly from such an array:
```

Comment:
I think I add the AnalysisFromFunction example in the section of the release notes where we introduce the feature. 

@orbeckst you mentioned to simulate a x-ray example by adding gaussian noise. Would that be added per frame? Because adding them per frame can be done with a small modification to the above AnalysisFromFunction code.

```python
coordinates = AnalysisFromFunction(lambda ag: ag.positions.copy() + np.random.normal(size=ag.positions.shape), 
                                   u.trajectory, u.atoms).run().results
```